_data/education.yml

```diff
@@ -117,6 +117,17 @@ websites:
       twitter: duolingo
       facebook: duolingo
       tfa: No
+      
+    - name: East Carolina University
+      url: https://www.ecu.edu
+      img: 3232eculogo.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      twitter: EastCarolina
+      facebook: EastCarolina
```

Comment:
No need for `twitter` and `facebook` if `tfa:yes`

Reply:
ha yeah i remade the logo in photoshop. let me try and shrink it

Reply:
Also please add `software: Yes` since your docs link also mentions they support software-based method.

Reply:
tfa = two factor auth right? I'm used to seeing this as 2fa.